package.json

```diff
@@ -25,11 +25,23 @@
     "stylelint": "stylelint **/*.scss",
     "test": "lerna run test",
     "typecheck": "lerna run typecheck",
-    "prepare": "lerna run prepare",
-    "postprepare": "[ -z \"$CI\" ] && husky install || exit 0"
+    "build": "lerna run build --ignore @okta/odyssey-ie-11-sandbox --ignore @okta/odyssey-storybook",
+    "postinstall": "yarn build && [ -z \"CI\" ] && husky install || exit 0"
   },
   "devDependencies": {
-    "lerna": "4.0.0"
+    "@commitlint/cli": "^14.0.0",
+    "@okta/odyssey-commitlint": "^0.11.2",
+    "@okta/odyssey-eslint-config": "^0.11.2",
+    "@okta/odyssey-lint-staged": "^0.11.2",
+    "@okta/odyssey-prettier": "^0.11.2",
+    "@okta/odyssey-stylelint": "^0.11.2",
+    "eslint": "^7.27.0",
+    "husky": "^7.0.2",
+    "lerna": "4.0.0",
+    "lint-staged": "^11.1.2",
+    "prettier": "^2.4.0",
+    "stylelint": "^13.13.1"
```

Comment:
I saw your note about opening an issue with the yarn repo about executables like this. As you know, previously we relied on these being hoisted to the monorepo root and made available. While it is probably good to be explicit, I don't love having to keep the version semver ranges in sync across two places (avoiding multiple installs or broken behavior across versions). Let me know what you find out please!

Reply:
Looks like this is the preferred solution. The alternative is to re-export the binaries... But that comes with its own can of worms. If other dependencies list the package as a peer dependency, then it will be listed as missing. So for the time being we'll have to define peer dependencies.

Reply:
awesome thanks @michaeltamaki-okta